shelley/chain-and-ledger/formal-spec/crypto-primitives.tex

```diff
@@ -85,10 +85,26 @@ \section{Cryptographic primitives}
 \end{figure}
 
 When we get to the blockchain layer validation, we will use
-key evolving signatures according to the MMM scheme \citep{cryptoeprint:2001:034}.
-Figure~\ref{fig:kes-defs-shelley} introduces the additional cryptographic abstractions
+key evolving signatures (KES) according to the MMM scheme \citep{cryptoeprint:2001:034}.
+This is another asymmetric key cryptographic scheme, also relying on
+the use of public and private key pairs.
+These signature schemes provide forward cryptographic security, meaning that a
+compromised key does not make it easier for an adversary to forge a signature that
+allegedly had been signed in the past.
+Figure~\ref{fig:crypto-defs-shelley} introduces the additional cryptographic abstractions
 needed for KES.
 
+In KES, the public verification key stays constant, but the
+corresponding private key evolves incrementally. For this reason, KES
+verification keys are indexed by integers representing the step in the key's
+evolution. This evolution step parameter is also an additional parameter needed
+for the signing (denoted by $\signEv$) and verification
+(denoted by $\verifyEv$) functions.
+
+Since the private key evolves incrementally in a KES scheme, the ledger rules
+must dictate the amount of time between incrementations. This is referred to as
```

Comment:
another way to say this is that the ledger is requiring the pool operators to evolve their keys once per `KESPeriod`-many slots.